test/performance/utils.js

```diff
@@ -2,12 +2,56 @@
  * External dependencies
  */
 import path from 'path';
-import { existsSync, readFileSync, unlinkSync, writeFileSync } from 'fs';
+import { existsSync, readFileSync, unlinkSync } from 'fs';
+
+export function sum( array ) {
+	if ( ! array || ! array.length ) return undefined;
```

Comment:
Nit: For testing I think it might be better to just throw an error if we're passing unexpected arguments.

Reply:
I think the error throwing should happen one step earlier (where we collect results data, for example, via `expect()`). Before that, though, I wanted to migrate the tests to TypeScript (in a separate PR) to get a better feel for how to handle exceptions.

For these calc functions (sum, median, average, etc.) I think it's OK not to throw, but it's still important to stay consistent in how we handle invalid arguments (`return undefined`). Having said that, I'm aware this implementation is not super-defensive, but I don't think it's necessary at this point.

Reply:
If we were using TypeScript, we wouldn't have to guard the inputs during runtime since the error would be caught statically. 👍